runtime/libtapasco/src/device.rs

```diff
@@ -221,6 +221,7 @@ pub struct Device {
     name: String,
     access: tlkm_access,
     scheduler: Arc<Scheduler>,
+    #[get = "pub"]
```

Comment:
This should not be necessary, better use a proper API to access the platform memory. Namely in this case most likely `get_platform_component_memory` and `get_available_platform_components`. I know this is still in progress, so this is just for future reference before a possible merge.

Reply:
Thanks. I've cherry-picked the respective commit from your PR and adapted the code to use `get_platform_component_memory`.